src/main/resources/db/migration/V1_1__create_relations.sql

```diff
@@ -0,0 +1,148 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE UserEntity (
```

Comment:
these things need to be named as they are in Kotlin (or I have a lot of fun with configuration of the library)
in the code, I generally try to avoid naming classes the same, even if they are in different packages (wrong imports etc.)
however, I had to name the GraphQL entities how they should be named in the schema if I want to avoid having to apply `GraphQLName` to everything
thus I decided this suffix is the easiest solution